pkgs/tools/misc/vector/default.nix

```diff
@@ -2,42 +2,56 @@
 , lib
 , fetchFromGitHub
 , rustPlatform
-, openssl
+  # nativeBuildInputs
 , pkg-config
+  # buildInputs
+, openssl
 , protobuf
+, rdkafka
+, oniguruma
+, zstd
+  # darwin buildInputs
```

Comment:
```suggestion
, pkg-config
, openssl
, protobuf
, rdkafka
, oniguruma
, zstd
```

Reply:
You don't like my comments? too much noise?
I'm fine with removing them.